src/chocolatey/GetChocolatey.cs

```diff
@@ -49,8 +49,9 @@ public class GetChocolatey
 
         public GetChocolatey()
         {
+            Log4NetAppenderConfiguration.configure();

```

Comment:
It's set by a call to Bootstrap.initialize - https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure/registration/Bootstrap.cs#L36

But then for some reason I am also erroneously setting it in ContainerBinding (where you get it) - https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure.app/registration/ContainerBinding.cs#L45

I need to fix that.